packages/common/src/pipes/slice_pipe.ts

```diff
@@ -61,12 +61,13 @@ export class SlicePipe implements PipeTransform {
    *   - **if positive**: return all items before `end` index of the list or string.
    *   - **if negative**: return all items before `end` index from the end of the list or string.
    */
+  transform<T>(value: ReadonlyArray<T>|null|undefined, start: number, end?: number): Array<T>|null;
   transform<T>(value: ReadonlyArray<T>, start: number, end?: number): Array<T>;
+  transform(value: string|null|undefined, start: number, end?: number): string|null;
   transform(value: string, start: number, end?: number): string;
-  transform(value: null, start: number, end?: number): null;
-  transform(value: undefined, start: number, end?: number): undefined;
-  transform(value: any, start: number, end?: number): any {
-    if (value == null) return value;
+  transform<T>(value: ReadonlyArray<T>|string|null|undefined, start: number, end?: number):
+      Array<T>|string|null {
+    if (value == null) return null;
```

Comment:
Prefer strict equality

```suggestion
    if (value === null || value === undefined) return null;
```

it's a nit but it makes the intention clearer IMO

Reply:
Right now all of the (common) pipes that work on scalar values check for `== null`. Should I change it also in the other ones?

Reply:
didn’t realize it was a convention, disregard